contracts/BC_fusion/StakeHub.sol

```diff
@@ -0,0 +1,786 @@
+            transferGasLimit = newTransferGasLimit;
+        } else if (key.compareStrings("minSelfDelegationBNB")) {
+            require(value.length == 32, "length of minSelfDelegationBNB mismatch");
+            uint256 newMinSelfDelegationBNB = value.bytesToUint256(32);
+            require(newMinSelfDelegationBNB >= 1000 ether, "the minSelfDelegationBNB is out of range");
+            newMinSelfDelegationBNB = newMinSelfDelegationBNB;
+        } else if (key.compareStrings("minDelegationBNBChange")) {
+            require(value.length == 32, "length of minDelegationBNBChange mismatch");
+            uint256 newMinDelegationBNBChange = value.bytesToUint256(32);
+            require(newMinDelegationBNBChange >= 1 ether, "the minDelegationBNBChange is out of range");
+            minDelegationBNBChange = newMinDelegationBNBChange;
+        } else if (key.compareStrings("maxElectedValidators")) {
+            require(value.length == 32, "length of maxElectedValidators mismatch");
+            uint256 newMaxElectedValidators = value.bytesToUint256(32);
+            require(newMaxElectedValidators >= 1, "the maxElectedValidators is out of range");
+            maxElectedValidators = newMaxElectedValidators;
+        } else if (key.compareStrings("unbondPeriod")) {
+            require(value.length == 32, "length of unbondPeriod mismatch");
+            uint256 newUnbondPeriod = value.bytesToUint256(32);
+            require(newUnbondPeriod >= 3 days, "the unbondPeriod is out of range");
+            unbondPeriod = newUnbondPeriod;
+        } else if (key.compareStrings("downtimeSlashAmount")) {
+            require(value.length == 32, "length of downtimeSlashAmount mismatch");
+            uint256 newDowntimeSlashAmount = value.bytesToUint256(32);
+            require(
+                newDowntimeSlashAmount >= 5 ether && newDowntimeSlashAmount < doubleSignSlashAmount,
+                "the downtimeSlashAmount is out of range"
+            );
+            downtimeSlashAmount = newDowntimeSlashAmount;
+        } else if (key.compareStrings("doubleSignSlashAmount")) {
+            require(value.length == 32, "length of doubleSignSlashAmount mismatch");
+            uint256 newDoubleSignSlashAmount = value.bytesToUint256(32);
+            require(
+                newDoubleSignSlashAmount >= 1000 ether && newDoubleSignSlashAmount > downtimeSlashAmount,
+                "the doubleSignSlashAmount is out of range"
+            );
+            doubleSignSlashAmount = newDoubleSignSlashAmount;
+        } else if (key.compareStrings("downtimeJailTime")) {
+            require(value.length == 32, "length of downtimeJailTime mismatch");
+            uint256 newDowntimeJailTime = value.bytesToUint256(32);
+            require(
+                newDowntimeJailTime >= 2 days && newDowntimeJailTime < doubleSignJailTime,
+                "the downtimeJailTime is out of range"
```

Comment:
I think in this PR, the code style of error message is not unified, sometimes use "UPPER SHORT" wording, sometime like this.

I prefer the "UPPER SHORT" type. please unify it.

Reply:
Done